package.json

```diff
@@ -95,25 +92,31 @@
     "codecov": "^3.0.2",
     "concat-stream": "^2.0.0",
     "dedent": "^0.7.0",
+    "duplexify": "^4.1.1",
     "eslint": "^6.0.0",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-node": "^11.0.0",
     "eslint-plugin-prettier": "^3.0.0",
-    "gts": "^1.0.0",
+    "execa": "^4.0.0",
+    "gts": "2.0.0-alpha.9",
```

Comment:
@JustinBeckwith if you give me broad strokes of what's needed, I might be able to help draft the migration guide.